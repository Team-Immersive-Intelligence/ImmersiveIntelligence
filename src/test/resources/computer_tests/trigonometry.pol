use ADVANCED_ARITHMETIC ;here also a comment!
use TEXT
use TYPE_CONVERSION
use COMPARATOR
use ARRAY

integer i = 0

type "This program will display sin values for x 0 to 45"

mark f

float e = sin @i
;round to 4 places
e =

type string_join (string_join (string_join "sin " (<string> @i)) " = ") (<string> @e) ;cos i = ...
integer i = + @i 1

array a = array_push @a @e ;push to array

wait 3

if < @i 46
goto "f"

type string_join "Array: " (<string> @a) ;print array
type "Koniec Roboty"

end