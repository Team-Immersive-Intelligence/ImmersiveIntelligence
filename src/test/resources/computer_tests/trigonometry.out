[POL] This program will display sin values for x 0 to 45
[POL] sin 0 = 0.0
[POL] sin 1 = 0.017452406
[POL] sin 2 = 0.034899496
[POL] sin 3 = 0.052335955
[POL] sin 4 = 0.06975647
[POL] sin 5 = 0.087155744
[POL] sin 6 = 0.104528464
[POL] sin 7 = 0.12186934
[POL] sin 8 = 0.1391731
[POL] sin 9 = 0.15643446
[POL] sin 10 = 0.17364818
[POL] sin 11 = 0.190809
[POL] sin 12 = 0.20791169
[POL] sin 13 = 0.22495106
[POL] sin 14 = 0.2419219
[POL] sin 15 = 0.25881904
[POL] sin 16 = 0.27563736
[POL] sin 17 = 0.2923717
[POL] sin 18 = 0.309017
[POL] sin 19 = 0.32556817
[POL] sin 20 = 0.34202015
[POL] sin 21 = 0.35836795
[POL] sin 22 = 0.37460658
[POL] sin 23 = 0.39073113
[POL] sin 24 = 0.40673664
[POL] sin 25 = 0.42261827
[POL] sin 26 = 0.43837115
[POL] sin 27 = 0.4539905
[POL] sin 28 = 0.46947157
[POL] sin 29 = 0.4848096
[POL] sin 30 = 0.5
[POL] sin 31 = 0.5150381
[POL] sin 32 = 0.52991927
[POL] sin 33 = 0.54463905
[POL] sin 34 = 0.5591929
[POL] sin 35 = 0.57357645
[POL] sin 36 = 0.58778524
[POL] sin 37 = 0.60181504
[POL] sin 38 = 0.6156615
[POL] sin 39 = 0.6293204
[POL] sin 40 = 0.64278764
[POL] sin 41 = 0.656059
[POL] sin 42 = 0.6691306
[POL] sin 43 = 0.6819984
[POL] sin 44 = 0.6946584
[POL] sin 45 = 0.70710677
[POL] Array: []
[POL] Koniec Roboty
